client-management-service-impl/src/main/java/io/mosip/esignet/services/ClientManagementServiceImpl.java

```diff
@@ -90,31 +88,10 @@ public ClientDetailResponse createOIDCClient(ClientDetailCreateRequest clientDet
         clientDetail.setStatus(CLIENT_ACTIVE_STATUS);
         clientDetail.setCreatedtimes(LocalDateTime.now(ZoneId.of("UTC")));
 
-        try {
-            clientDetail = clientDetailRepository.save(clientDetail);
-        } catch (ConstraintViolationException cve) {
-            log.error("Failed to create client details", cve);
-            throw new EsignetException(ErrorConstants.DUPLICATE_PUBLIC_KEY);
-        }
-
-        auditWrapper.logAudit(AuditHelper.getClaimValue(SecurityContextHolder.getContext(), claimName),
-        		Action.OIDC_CLIENT_CREATE, ActionStatus.SUCCESS, AuditHelper.buildAuditDto(clientDetailCreateRequest.getClientId()), null);
-
-        var response = new ClientDetailResponse();
-        response.setClientId(clientDetail.getId());
-        response.setStatus(clientDetail.getStatus());
-        return response;
+        return clientDetail;
     }
 
-    @CacheEvict(value = Constants.CLIENT_DETAIL_CACHE, key = "#clientId")
-    @Override
-    public ClientDetailResponse updateOIDCClient(String clientId, ClientDetailUpdateRequest clientDetailUpdateRequest) throws EsignetException {
-        Optional<ClientDetail> result = clientDetailRepository.findById(clientId);
-        if (!result.isPresent()) {
-            throw new EsignetException(ErrorConstants.INVALID_CLIENT_ID);
-        }
-
-        ClientDetail clientDetail = result.get();
+    private ClientDetail toClientDetailDTO(ClientDetail clientDetail, ClientDetailUpdateRequest clientDetailUpdateRequest) {
```

Comment:
This is an overloaded method, change this method name also to buildClientDetailEntity